src/main/java/org/folio/uk/controller/UserController.java

```diff
@@ -28,7 +29,7 @@ public ResponseEntity<User> createUser(User user, Boolean keycloakOnly) {
 
   @Override
   public ResponseEntity<User> getUser(UUID id) {
-    return ResponseEntity.ok(service.getUser(id));
+    return ResponseEntity.ok(service.getUser(id).orElseThrow(() -> new EntityNotFoundException("Not Found")));
```

Comment:
This method is aimed to retrieve user information from mod-users and has different url `/users-keycloak/users/{id}.` It has nothing to do with Keycloak users (auth users).
You have to add another method

Reply:
I know it has nothing to do with auth users - I didn't change anything in that regard here. All I changes was to use Optional instead of returning null.

Reply:
Yes, sorry my mistake. Resolving this comment..